scripts/jira_tickets/create_compara_release_JIRA_tickets.pl

```diff
@@ -91,8 +91,8 @@ sub usage {
 -division | -d       Compara division. Optional, will be obtained from environment variable \$COMPARA_DIV
                      if not supplied.
 -tickets             File that holds the input data for creating the JIRA tickets in JSON format. Optional,
-                     the script will look for 'jira_recurrent_tickets.<division>.json' in the same
-                     directory as the executable if not supplied.
+                     the script will look for '<division>/jira_recurrent_tickets.json' in the 'conf' directory
+                     two levels above the executable if not supplied.
```

Comment:
If the code is changed based on my previous comment, this should be changed accordingly.